.travis.yml

```diff
@@ -2,49 +2,40 @@ branches:
   except:
   - appveyor
 
-language: c
-
-matrix:
-  include:
-    - os: linux
-      dist: trusty
-      sudo: required
-      services: docker
-      env: IMAGE=stretch+mcode
-    - os: linux
-      dist: trusty
-      sudo: required
-      services: docker
-      env: IMAGE=stretch+mcode+gpl
-    - os: linux
-      dist: trusty
-      sudo: required
-      services: docker
-      env: IMAGE=ubuntu14+mcode
-    - os: linux
-      dist: trusty
-      sudo: required
-      services: docker
-      env: IMAGE=ubuntu14+llvm-3.8
-    - os: linux
-      dist: trusty
-      sudo: required
-      services: docker
-      env: IMAGE=fedora26+mcode
-    - os: osx
-      osx_image: xcode7.3
-      env: IMAGE=macosx+mcode
-      cache:
-        directories:
-          - gnat
+language: minimal
 
+os: linux
+dist: trusty
+sudo: required
+services: docker
 install: true
 
-script:
-- ./dist/linux/travis-ci.sh
+addons:
+  apt:
+    packages:
+      - docker-ce
```

Comment:
Do you need docker-ce for this patch ?

Reply:
Not really, sorry. It is required for multistage builds (#489). I'll remove it from here.